docs/unified/novacustom/recovery.md

```diff
@@ -81,102 +141,283 @@ make BOARD=arduino/mega2560
 make BOARD=arduino/mega2560 flash
 ```
 
-#### Flashing
+### Recovery steps
 
-- Unscrew the bottom cover from the laptop
+#### V560
```

Comment:
Where are the recovery steps for v54? Are they the same as for V56? We should indicate that if they are.

Reply:
This paragraph is for V560TU only because it's the only model with the keyboard glued to the chassis. All other models should skip this step

Reply:
Okay, this makes sense, but it wasn't clear to me. Can we add this note between 'Recovery Steps' and the 'V560' headings @dawidlesiak.